### PR TITLE
Spanish translation for Cohen's d

### PR DESCRIPTION
**Pull request by danalclop**
Current progress:

```bash
M        [X] gatsby-theme-rpsych/assets/availableTranslations.yml

M        [X] gatsby-theme-rpsych/i18n/config.json

A        [X] gatsby-theme-rpsych/i18n/react-i18next/<lang_code>/404.json
A        [X] gatsby-theme-rpsych/i18n/react-i18next/<lang_code>/blog.json

A        [X] gatsby-theme-rpsych-cohend/i18n/react-i18next/<lang_code>/cohend.json

A        [X] gatsby-theme-rpsych-cohend/content/cohend/intro/intro.<lang_code>.mdx

A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/0_help/0_help.<lang_code>.md
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/1_formulas/1_formulas.<lang_code>.md
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/2_citation/2_citation.<lang_code>.mdx
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/3_bug/3_bug.<lang_code>.md
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/4_load/4_load.<lang_code>.md
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/5_overlap/5_overlap.<lang_code>.md
A        [X] gatsby-theme-rpsych-cohend/content/cohend/FAQ/6_viz_license/6_viz_license.<lang_code>.md

A        [X] gatsby-theme-rpsych-cohend/content/cohend/common-language/common-language.<lang_code>.mdx
```

### CODE REVIEW

**Comment by danalclop** [comment]
There is a small issue with the Spanish translation of the file `cohend.json` at `gatsby-theme-rpsych-cohend/i18n/react-i18next/es/`

The phrase: "Interpreting Cohen's" (omitting the *d*) can not be directly translated to Spanish, since the Spanish equivalent to the English possessive ([NOUN]*’s*) is placed **before** the noun (*de* [NOUN]); thus, you won't be able to pipe the key "Interpreting Cohen's" with another key or element after:

	[EN] Interpreting Cohen's [d]

	[ES] Interpretando la [d] de Cohen


Provisionally, I translated that key with the complete phrase: "Interpreting Cohen's d"  (including the *d*).  Tell me what you think about it and whether we can find a workaround.

**Comment by danalclop** [comment]
I tried building the docker container to preview the site locally but something failed (got an `error Command "gatsby" not found` error message followed by an `Exit code: 1` message). I'm not experienced in Docker, would you mind sending me a preview link so I can check everything's fine?

**Comment by rpsychologist** [comment]
There's a problem with these files
- gatsby-theme-rpsych/i18n/react-i18next/es/blog.json 
- gatsby-theme-rpsych/i18n/react-i18next/es/404.json 

They are complete HTML files :)

> There is a small issue with the Spanish translation of the file `cohend.json` at `gatsby-theme-rpsych-cohend/i18n/react-i18next/es/`
> 
> The phrase: "Interpreting Cohen's" (omitting the _d_) can not be directly translated to Spanish, since the Spanish equivalent to the English possessive ([NOUN]_’s_) is placed **before** the noun (_de_ [NOUN]); thus, you won't be able to pipe the key "Interpreting Cohen's" with another key or element after:
> 
> ```
> [EN] Interpreting Cohen's [d]
> 
> [ES] Interpretando la [d] de Cohen
> ```
> 
> Provisionally, I translated that key with the complete phrase: "Interpreting Cohen's d" (including the _d_). Tell me what you think about it and whether we can find a workaround.

Thanks, I've fixed this and some other keys that could possible cause problems in the future

**Comment by danalclop** [comment]
Done! Don't know what happened, I simply copied the original files, translated, and saved. Now they look fine for me 👍

File: gatsby-theme-rpsych-cohend/content/cohendTranslators.yml [hunk 1]
```diff
@@ -8,9 +8,9 @@
 - lang: es
   translator: 
     -
-      name: Translator Name ES
-      url: https://translator-website.se
-      twitter: krs3
+      name: Daniel Alcalá
+      url: https://temp.es
+      twitter: temp
```
Comment (rpsychologist) [inline gatsby-theme-rpsych-cohend/content/cohendTranslators.yml:13]: I've added a new component. Update this if you want to be acknowledged :)

File: gatsby-theme-rpsych-cohend/i18n/react-i18next/es/cohend.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
-  "Interpreting Cohen's": "Interpretando la d de Cohen",
   "effect size": "tamaño del efecto",
   "Cohen's": "de Cohen",
   "Cohen's d": "La d de Cohen",
+  "CohenLabelSlider": "La <i>d</i> de Cohen",
+  "CohenU3": "La U<1>3</1> de Cohen",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych-cohend/i18n/react-i18next/es/cohend.json:6]: Added new keys, I guess this is correct?

File: gatsby-theme-rpsych/i18n/react-i18next/es/blog.json [hunk 3]
```diff
@@ -20,7 +19,8 @@
   "Save settings in browser": "Guardar los ajustes en el navegador",
   "Saving failed": "El guardado falló",
   "Clear": "Limpiar",
-  "authorBio": "Escrito por <strong>Kristoffer Magnusson</strong>, investigador en psicología clínica. <Puedes seguirle en Twitter</4> y pasarte por el discord de ciencia abierta <8>Git Gud Science</8>.",
+  "authorBio": "Escrito por <strong>Kristoffer Magnusson</strong>, investigador en psicología clínica. <4>Puedes seguirle en Twitter</4> y pasarte por el discord de ciencia abierta <8>Git Gud Science</8>.",
+  "Translated by": "Traducido por",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych/i18n/react-i18next/es/blog.json:23]: New key, is this correct?

File: gatsby-theme-rpsych/i18n/react-i18next/es/blog.json [hunk 4]
```diff
@@ -7,9 +7,8 @@
   "Webmentions sent before 2021": "(Lamentablemente, las menciones web enviadas antes de 2021 no aparecerán aquí.)",
   "There are no webmentions for this page": "No hay webmentions de esta página",
   "Designed and built by": "Diseñado y elaborado por Kristoffer Magnusson. Con tecnología",
-  "bought": "ha comprado",
-  "coffee": "café",
-  "coffees": "cafés",
+  "coffeeSupporters": "<0>{{name}}</0> ha comprado {{coffeeSymbols}} ({{count}}) café",
+  "coffeeSupporters_plural": "<0>{{name}}</0> ha comprado {{coffeeSymbols}} ({{count}}) cafés",
```
Comment (rpsychologist) [inline gatsby-theme-rpsych/i18n/react-i18next/es/blog.json:11]: I reorganized this

**Review by rpsychologist — COMMENTED** [review]
I've made some minor tweaks, can you review the marked changes?

I've also deployed a preview here https://602ac32be4ec295a7c59bdb0--rpsychologist.netlify.app/es/cohend/

**Comment by danalclop** [comment]
Okay, I've gone through everything paying attention to how it will look in the preview and it all looks good to me 👏

**Comment by rpsychologist** [comment]
Thanks! I really appreciate the help @danalclop. The translation is live now

**Pull request merged**